### PR TITLE
CA Template - Docker Virt Manager

### PR DESCRIPTION
**Pull request by interbiznw**

### CODE REVIEW

**Comment by jrdnlc** [comment]
Shaving off the XML is needed before merging. Read [here](https://selfhosters.net/docker/templating/templating/)

**Comment by interbiznw** [comment]
> Shaving off the XML is needed before merging. Read [here](https://selfhosters.net/docker/templating/templating/)

Thought I already got most of it, what specific thing is still lingering? Thanks!

**Comment by interbiznw** [comment]
Sorry, Think its all cleaned up now. Thanks!

File: templates/docker-virt-manager.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <Network>bridge</Network>
+  <Shell>bash</Shell>
+  <Privileged>true</Privileged>
+  <Support>https://forums.unraid.net/topic/77603-virt-manager-intel-gpu-tools-and-more-dockers/</Support>
+  <Project>https://virt-manager.org/</Project>
+  <Overview>GTK Broadway web UI for libvirt. The virt-manager application is a desktop user interface for managing virtual machines through libvirt. It primarily targets KVM VMs, but also manages Xen and LXC (linux containers). It presents a summary view of running domains, their live performance &amp;amp; resource utilization statistics. Wizards enable the creation of new domains, and configuration &amp;amp; adjustment of a domain&#x2019;s resource allocation &amp;amp; virtual hardware. Uses GTK3 Broadway (HTML5) backend--no vnc, xrdp, etc needed! Also has DARK MODE</Overview>
+  <Category>Tools:</Category>
+  <Beta>true</Beta>
+  <WebUI>http://[IP]:[PORT:8185]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/UnRaid/template/docker-virt-manager.xml</TemplateURL>
```
Comment (Roxedus) [inline templates/docker-virt-manager.xml:15]: No

File: templates/docker-virt-manager.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <Icon>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/virt-manager-icon.png</Icon>
+  <DonateText>If you like my work please consider Donating.</DonateText>
+  <DonateLink>https://interbiznw.github.io/</DonateLink>
```
Comment (Roxedus) [inline templates/docker-virt-manager.xml:18]: What work?

File: templates/docker-virt-manager.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  <Config Name="KVM" Target="/dev/kvm" Default="" Mode="" Description="Container Path: /dev/kvm" Type="Device" Display="always" Required="false" Mask="false">/dev/kvm</Config>
+  <Config Name="/var/run/libvirt/libvirt-sock" Target="/var/run/libvirt/libvirt-sock" Default="" Mode="rw" Description="Container Path: /var/run/libvirt/libvirt-sock" Type="Path" Display="always" Required="false" Mask="false">/var/run/libvirt/libvirt-sock</Config>
+  <Config Name="/var/lib/libvirt/images" Target="/var/lib/libvirt/images" Default="" Mode="rw" Description="Container Path: /var/lib/libvirt/images" Type="Path" Display="always" Required="false" Mask="false">/var/lib/libvirt/images</Config>
```
Comment (Roxedus) [inline templates/docker-virt-manager.xml:22]: These looks like they should all be hidden under advanced view.

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/docker-virt-manager.xml (hunk 1 above)
Comment (interbiznw) [reply templates/docker-virt-manager.xml:15]: no what?

File: templates/docker-virt-manager.xml (hunk 2 above)
Comment (interbiznw) [reply templates/docker-virt-manager.xml:18]: Creating the template and getting a new functional and better UI/UX Virt manager in UNRAID again.

File: templates/docker-virt-manager.xml (hunk 3 above)
Comment (interbiznw) [reply templates/docker-virt-manager.xml:22]: Sorry, new to templates. Will check that out and change.

File: templates/docker-virt-manager.xml (hunk 2 above)
Comment (Roxedus) [reply templates/docker-virt-manager.xml:17]: I see nothing pointing to the fact that it's your image.

File: templates/docker-virt-manager.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Docker-Virt-Manager</Name>
+  <Repository>mber5/virt-manager:latest</Repository>
+  <Registry>https://hub.docker.com/r/mber5/virt-manager</Registry>
+  <Network>bridge</Network>
+  <Privileged>true</Privileged>
+  <Support>https://forums.unraid.net/topic/77603-virt-manager-intel-gpu-tools-and-more-dockers/</Support>
```
Comment (Roxedus) [inline templates/docker-virt-manager.xml:8]: The author of this thread clearly don't want to be associated with this.

File: templates/docker-virt-manager.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  <Project>https://virt-manager.org/</Project>
+  <Overview>GTK Broadway web UI for libvirt. The virt-manager application is a desktop user interface for managing virtual machines through libvirt. It primarily targets KVM VMs, but also manages Xen and LXC (linux containers). It presents a summary view of running domains, their live performance &amp;amp; resource utilization statistics. Wizards enable the creation of new domains, and configuration &amp;amp; adjustment of a domain&#x2019;s resource allocation &amp;amp; virtual hardware. Uses GTK3 Broadway (HTML5) backend--no vnc, xrdp, etc needed! Also has DARK MODE</Overview>
+  <Category>Tools:</Category>
+  <Beta>true</Beta>
+  <WebUI>http://[IP]:[PORT:8185]</WebUI>
```
Comment (Roxedus) [inline templates/docker-virt-manager.xml:13]: PORT should reference the internal port.

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by interbiznw** [comment]
Fixed

File: templates/docker-virt-manager.xml (hunk 2 above)
Comment (interbiznw) [reply templates/docker-virt-manager.xml]: Not sure what difference that makes, I took the time to create the template that apparently no one has created for this image...but apparently that is meaningless rite? Cool. Removed.

Comment (interbiznw) [reply templates/docker-virt-manager.xml]: Next time Ill just copy the image and put it in my own dockerhub 😂

**Pull request closed without merging**